MessengerProj/src/main/res/values/strings.xml

```diff
@@ -329,11 +329,11 @@
     <string name="AskStartChatWith">Chat with <![CDATA[<b>]]>%1$s<![CDATA[</b>]]>?</string>
     <string name="DeaddropHint">Tap the reply-arrows to start chatting.</string>
     <string name="NotYetImplemented">This function is not available or incomplete.</string>
-    <string name="DefaultStatusText">Sent with my Delta Chat Messenger: https://delta.chat</string>
+    <string name="DefaultStatusText">Corresponded under the aegis of Delta Chat Messenger: https://delta.chat</string>
```

Comment:
?

Reply:
Meaning: Under the protection, sponsorship, or patronage of; under the supervision, control, or management of.
"Sent with" is the mark of uncanny comparison.